content/docs/user-guide/project-structure/configuration.md

```diff
@@ -378,6 +379,22 @@ Composition].
 
 <details>
 
+## studio
+
+- `studio.token` - Studio access token to use. When this is set, DVC uses this
+  to notify Studio of new experiments, and provide live updates of metrics and
+  plots to Studio. Alternatively, this can also be specified through
+  `DVC_STUDIO_TOKEN` environment variable. For security reasons, we advise
+  setting token to either a local config or a global config. Note that the above
+  functionality is disabled if the config or the environment variable is not
+  set.
```

Comment:
What does the last sentence mean? Is it needed?

Reply:
re-emphasizing?

Reply:
I removed last sentence.